src/tasks.js

```diff
@@ -0,0 +1,103 @@
+ko.tasks = (function () {
```

Comment:
Also, if we want the license statement in the minified file, it will need to be a string instead of a comment.